style.css

```diff
@@ -102,105 +233,193 @@ main {
   line-height: 24px;
 }
 
-main #headline ul {
-  list-style-type: none;
-}
-
-main #headline ul li {
-  display: inline-flex;
-  padding: 0 4px;
-  margin: 10px 0;
-  flex-direction: column;
-  align-items: flex-start;
-  gap: 12px;
+#about-myself h3 {
+  color: var(--color4);
+  text-transform: uppercase;
+  font-family: 'Poppins', sans-serif;
   font-size: 16px;
-}
-
-#headline ul li a:last-child {
-  padding-left: 0;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  padding: 10px 0;
 }
 
 #headline h3:hover {
   color: var(--primary-1);
 }
 
-main .card-works {
+/* .card-works {
+  width: 90%;
+  margin: auto;
   display: grid;
   grid-template-rows: repeat(1, 1fr);
-  grid-template-columns: repeat(1fr);
   margin-top: 120px;
   gap: 50px;
-  padding: 14px;
+  padding: 20px;
   background-color: var(--white);
   border-radius: 16px;
-  margin-left: 25px;
-  margin-right: 25px;
-  border: 1px solid #e7e7e7;
-}
+  border: 1px solid var(--color12);
+} */
```

Comment:
- [ ] It's a good thing you currently load this part of your portfolio dynamically. How about removing the whole of this completely? Commenting old pieces of code in a workspace makes the code looks kinda messy :hammer_and_wrench: